sass/components/html/listing__wrapper--row-three.html

```diff
@@ -1,7 +1,7 @@
 <div class="listing__wrapper listing__wrapper--row-three">
   <ul class="listing listing--no-border listing--teaser listing--cta">
     <li class="listing__item">
-      <div class="listing__item__wrapper">
```

Comment:
I think for the example we should keep the class name `listing__item__wrapper`, because we have it defined in the rules.